### PR TITLE
Expose refresh policy

### PR DESCRIPTION
**Pull request by liatiusim**

### CODE REVIEW

File: persistence-spi/src/main/java/org/apache/unomi/persistence/spi/PersistenceService.java [hunk 1]
```diff
@@ -79,6 +79,14 @@
      */
     <T extends Item> PartialList<T> getAllItems(final Class<T> clazz, int offset, int size, String sortBy, String scrollTimeValidity);
 
+    /**
+     * Return true if the item which is saved in the persistence service is consistent
+     *
+     * @param itemType the item name
+     * @return {@code true} if the item is consistent, false otherwise
+     */
+    boolean isConsistnet(String itemType);
```
Comment (lyogev) [inline persistence-spi/src/main/java/org/apache/unomi/persistence/spi/PersistenceService.java:88]: Consistent

Comment (lyogev) [reply persistence-spi/src/main/java/org/apache/unomi/persistence/spi/PersistenceService.java:88]: Maybe it should get the item itself?

File: package/src/main/resources/etc/custom.system.properties [hunk 2]
```diff
@@ -67,6 +67,7 @@ org.apache.unomi.elasticsearch.cluster.name=${env:UNOMI_ELASTICSEARCH_CLUSTERNAM
 # hostA:9200,hostB:9200
 # Note: the port number must be repeated for each host.
 org.apache.unomi.elasticsearch.addresses=${env:UNOMI_ELASTICSEARCH_ADDRESSES:-localhost:9200}
+org.apache.unomi.elasticsearch.itemTypeToRefreshPolicy=${env:UNOMI_ELASTICSEARCH_REFRESH_POLICY_PER_ITEM_TYPE:-}
```
Comment (lyogev) [inline package/src/main/resources/etc/custom.system.properties:70]: Can you add an example of what can be added here?

File: persistence-elasticsearch/core/src/main/java/org/apache/unomi/persistence/elasticsearch/ElasticSearchPersistenceServiceImpl.java [hunk 3]
```diff
@@ -215,6 +215,13 @@ public void setElasticSearchAddresses(String elasticSearchAddresses) {
         }
     }
 
+    public void setItemTypeToRefreshPolicy(String itemTypeToRefreshPolicy) {
+        if(!itemTypeToRefreshPolicy.isEmpty()) {
```
Comment (lyogev) [inline persistence-elasticsearch/core/src/main/java/org/apache/unomi/persistence/elasticsearch/ElasticSearchPersistenceServiceImpl.java:219]: Can this be JSON and parsed with a JSON parser?

File: persistence-spi/src/main/java/org/apache/unomi/persistence/spi/PersistenceService.java (hunk 1 above)
Comment (liatiusim) [reply persistence-spi/src/main/java/org/apache/unomi/persistence/spi/PersistenceService.java:88]: Done

**Review by lyogev — APPROVED** [review]

**Pull request merged**